src/devtools/client/debugger/src/components/TestInfo/ContextMenu.module.css

```diff
@@ -5,6 +5,7 @@
   border-radius: 0.5rem;
   overflow: hidden;
   white-space: nowrap;
+  box-shadow: 0 4px 6px -1px rgb(0 0 0 / 0.1), 0 2px 4px -2px rgb(0 0 0 / 0.1);
```

Comment:
Seems like we should avoid adding more hard-coded colors?

Reply:
Thoughts on this?

Reply:
Here's where my head is ahead of our chat in 10 minutes:

1. I don't mind hard-coding variables in css files as a general rule, but this example reminded me that we do have lots of different drop-shadows
2. So I made a new class called dropshadow-md (because we do want different sizes across the app) -- but curious to hear your thoughts about that plan/naming convention

Reply:
> 1. I don't mind hard-coding variables in css files as a general rule,

I do oppose this, FWIW.

It's likely to cause problems with themes (light vs dark) and it's _very difficult_ to keep in sync (and/or update in bulk).

> 2. So I made a new class called dropshadow-md

This seems like the right direction.

Reply:
Following up: I'm just going to name this --dropshadow for now, and later I might add --dropshadow-large.